src/python/test/test_weighted_rips.py

```diff
@@ -0,0 +1,63 @@
+""" This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+    Author(s):       Yuichi Ike and Masatoshi Takenouchi
+
+    Copyright (C) 2020 Inria
+
+    Modification(s):
+      - YYYY/MM Author: Description of the modification
+"""
+
+from gudhi.weighted_rips_complex import WeightedRipsComplex
+from gudhi.point_cloud.dtm import DistanceToMeasure
+import numpy as np
+from math import sqrt
+from scipy.spatial.distance import cdist
+import pytest
+
+def test_non_dtm_rips_complex():
+    dist = [[], [1]]
+    weights = [1, 100]
+    w_rips = WeightedRipsComplex(distance_matrix=dist, weights=weights)
+    st = w_rips.create_simplex_tree(max_dimension=2)
+    assert st.filtration([0,1]) == pytest.approx(200.0)
+
+def test_compatibility_with_rips():
+    distance_matrix = [[0], [1, 0], [1, sqrt(2), 0], [sqrt(2), 1, 1, 0]]
+    w_rips = WeightedRipsComplex(distance_matrix=distance_matrix,max_filtration=42)
+    st = w_rips.create_simplex_tree(max_dimension=1)
+    assert list(st.get_filtration()) == [
+        ([0], 0.0),
+        ([1], 0.0),
+        ([2], 0.0),
+        ([3], 0.0),
+        ([0, 1], 1.0),
+        ([0, 2], 1.0),
+        ([1, 3], 1.0),
+        ([2, 3], 1.0),
+        ([1, 2], 1.4142135623730951),
```

Comment:
Maybe spell it `sqrt(2)` to be consistent with the input? Normally I'd want some approximate test, but since there is no actual computation here (just `f+0+0`) an exact test is ok.